### PR TITLE
HV-1895 Updated maven-bundle-plugin to 5.1.4

### PR DESCRIPTION
**Pull request by dmatej**
* https://hibernate.atlassian.net/browse/HV-1895
* Also updated several other properties to released artifacts
* Tested with GlassFish 7.0.0-SNAPSHOT

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -146,12 +146,12 @@
             They should be aligned with the ones from the Wildfly version we support
             See http://search.maven.org/#search|gav|1|g%3A"org.wildfly"%20AND%20a%3A"wildfly-parent"
         -->
-        <version.jakarta.enterprise.cdi-api>4.0.0-RC5</version.jakarta.enterprise.cdi-api>
-        <version.org.jboss.weld.weld>5.0.0.Beta1</version.org.jboss.weld.weld>
+        <version.jakarta.enterprise.cdi-api>4.0.0</version.jakarta.enterprise.cdi-api>
+        <version.org.jboss.weld.weld>5.0.0.CR1</version.org.jboss.weld.weld>
```
Comment (gsmet) [inline pom.xml:150]: I think CR2 is available so let's upgrade to CR2?

File: pom.xml [hunk 2]
```diff
@@ -146,12 +146,12 @@
             They should be aligned with the ones from the Wildfly version we support
             See http://search.maven.org/#search|gav|1|g%3A"org.wildfly"%20AND%20a%3A"wildfly-parent"
         -->
-        <version.jakarta.enterprise.cdi-api>4.0.0-RC5</version.jakarta.enterprise.cdi-api>
-        <version.org.jboss.weld.weld>5.0.0.Beta1</version.org.jboss.weld.weld>
+        <version.jakarta.enterprise.cdi-api>4.0.0</version.jakarta.enterprise.cdi-api>
```
[the remark this answers is not in the document]
Comment (dmatej) [reply pom.xml:149]: Oh, probably they are still waiting in staging repository ... can be added temporarily using simple profile.

**Comment by gsmet** [comment]
BTW, let me know if you need a new Alpha with these fixes, I will gladly push the button.

**Comment by dmatej** [comment]
I will run yet **_third_** build in the morning, now it glassfish builds with the previous hibernate-validator state in the PR:
https://ci.eclipse.org/glassfish/job/glassfish_build-and-test-using-jenkinsfile/job/PR-23880/
If it would pass, I would be very happy for the Alpha :-)

**Comment by gsmet** [comment]
OK, just ping me in the issue and I'll push the button.

**Comment by dmatej** [comment]
Good morning, and I say PING! You can push the button, thank you! :-)

**Comment by gsmet** [comment]
Tried a few times and got:
```
[ERROR] Remote staging finished with a failure: Failed to deploy artifacts: Could not transfer artifact org.hibernate.validator:hibernate-validator-test-utils:jar.asc:8.0.0.Alpha3 from/to ossrh ([https://oss.sonatype.org:443/service/local/staging/deployByRepositoryId/orghibernate-1253](https://oss.sonatype.org/service/local/staging/deployByRepositoryId/orghibernate-1253)): Failed to transfer file: [https://oss.sonatype.org:443/service/local/staging/deployByRepositoryId/orghibernate-1253/org/hibernate/validator/hibernate-validator-test-utils/8.0.0.Alpha3/hibernate-validator-test-utils-8.0.0.Alpha3.jar.asc](https://oss.sonatype.org/service/local/staging/deployByRepositoryId/orghibernate-1253/org/hibernate/validator/hibernate-validator-test-utils/8.0.0.Alpha3/hibernate-validator-test-utils-8.0.0.Alpha3.jar.asc). Return code is: 502, ReasonPhrase: Bad Gateway.
```

I'll try again later.

**Comment by gsmet** [comment]
@dmatej 8.0.0.Alpha3 has been released. Thanks for the fix!

**Comment by dmatej** [comment]
> @dmatej 8.0.0.Alpha3 has been released. Thanks for the fix!

Thanks for the quick cooperation! :-)

**Pull request merged**